modules/ringieraxelspringerAnalyticsAdapter.js

```diff
@@ -0,0 +1,15 @@
+import adapter from '../libraries/analyticsAdapter/AnalyticsAdapter.js';
+import adapterManager from '../src/adapterManager.js';
+
+var ringieraxelspringerAdapter = adapter({
+  global: 'ringieraxelspringerPrebidAnalytics',
+  handler: 'on',
+  analyticsType: 'bundle'
+});
+
+adapterManager.registerAnalyticsAdapter({
```

Comment:
am I out of date, don't you need an endpoint or something? How can an analytics adapter be this compact?

Reply:
I'll need to check in a moment, but this is what we were using up until now. I'll move these files to new PR as requested for now.